tests/test_lane.py

```diff
@@ -683,11 +687,21 @@ def test_groups(self):
 
     def test_featured_works(self):
         wl = MockWorks()
```

Comment:
Can this be called `work_list` instead of `wl`? It's tough to parse alongside `w1`.

Reply:
Although now I'm looking at the file, and it looks like wl is the common practice across tests. Feel free to leave it.